config/teensy_audio_board_rev_d.h

```diff
@@ -7,20 +7,34 @@
 
 enum SaberPins {
   // Bottom edge (in pin-out diagram)
+  // sdCardSelectPin is actually used, and can be altered if desired. 
+  // needs an external 10K pullup to 3.3V, provided on aduio shield
   sdCardSelectPin = 10,           // (See File->Example->SD->ReadWrite for
                                   // other possible values.)
-  bclkPin = 21,                   // BCLK (digital audio)
-  spiDataOut = 7,                 // spi out, serial flash, spi led & sd card
+
+  // these values are just a reminder not to use these pins for anything else, 
+  // they definitions are not actually used, and overwritten by the AUDIO_CONTROL_SGTL5000
+  spiDataOut = 11,                 // spi out, serial flash, spi led & sd card
   spiDataIn = 12,                 // spi in, serial flash & sd card
-  spiClock = 10,                  // spi clock, flash, spi led & sd card
+  spiClock = 13,                  // spi clock, flash, spi led & sd card
+
+  // these values are just a reminder not to use these pins for anything else, 
+  // they definitions are not actually used, and overwritten by the AUDIO_CONTROL_SGTL5000
+  i2cDataPin = 18,                // I2C bus, SGTL5000 control
+  i2cClockPin = 19,               // I2C bus, SGTL5000 control
 
-//  auxPin = 15,                    // AUX button
+//  auxPin = 15,                    // AUX button  ->> this is also Volume pin on audioshield, do not use
```

Comment:
the volume pin doesn't actually do anything, it's fine to use it.

Reply:
It will if you have the optional potentiometer mounted on the board, and it has a 0.1uF coupled do AGND permanently.
The stock shield, pin 15 can be re-used for other purposes

Reply:
Maybe make a note of that? When pot is installed do not use pin 15?

Reply:
Yes.
I would probably state it the other way, especially since ProffieOS doesn't use the pot by default.